nautobot/dcim/tests/test_api.py

```diff
@@ -1485,6 +1485,7 @@ class CableTest(APIViewTestCases.APIViewTestCase):
         "length": 100,
         "length_unit": "m",
     }
+    choices = ["termination_a_type", "termination_b_type", "type", "status", "length_unit"]
```

Comment:
So `choices` is a list of which fields should have choices, not what the values should be for those choices? `choices` is a bit of a vague name to me, perhaps something like `choices_fields` might be clearer?